package.json

```diff
@@ -61,6 +61,16 @@
           "type": "array",
           "default": [],
           "description": "Path(s) to files/folders that should be ignored when publishing diagnostics."
+        },
+        "septic.codeActions.insertEvrPosition": {
+          "scope": "window",
+          "type": "string",
+          "enum": [
+            "top",
+            "bottom"
+          ],
+          "default": "bottom",
+          "description": "Position of inserted evr. Bottom = Above CalcModl Top = Bellow DmmyAppl"
```

Comment:
"Evr" capitalised
"Below"
And while I'm at it: A comma or semi-colon after "CalcModl" wouldn't be wrong.